docs/telemetry.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_label: Telemetry
+title: Telemetry
+---
+
+Apify collects telemetry data about general usage of CLI to help us improve the product.
+Participation in this program is optional, and you may opt-out if you'd not like to share any information.
+
+## How is the data collected
+
+We currently collect the following metrics:
+
+- Usage of commands except `secrets` commands.
```

Comment:
I think we should track even the usage of the `secrets` command, but not the contents of the added secrets.

Plus we should also not track the `--token` flag on the `login` command.

Reply:
We should not track the contents of any parameter by default. Only the usage of parameters. I'd then manually whitelist the params to be tracked along with the value - i.e. selected template is ok to be tracked.